e2e-tests/playwright/e2e/plugins/rbac/rbac-api.spec.ts

```diff
@@ -23,9 +27,18 @@ test.describe("Test RBAC plugin REST API", () => {
     responseHelper = new Response(apiToken);
   });
 
-  test.beforeEach(
-    async () => await new Common(page).checkAndClickOnGHloginPopup(),
-  );
+  // eslint-disable-next-line no-empty-pattern
+  test.beforeEach(async ({}, testInfo) => {
+    try {
+      console.log(
+        `beforeEach: Attempting setup for ${testInfo.title}, retry: ${testInfo.retry}`,
+      );
+      await new Common(page).checkAndClickOnGHloginPopup();
```

Comment:
rbac test doesn't need github login. we may remove this.

Reply:
Not really sure if I'm understanding this comment right. Is this about something specifically related to GH login popup, or are you saying the user won't need to be logged in for these tests?
These tests focus specifically on a role created in the beginning for the rhdh-qe user.

    const members = ["user:default/rhdh-qe"];

    const newPolicy = {
      entityReference: "role:default/test",
      permission: "catalog-entity",
      policy: "read",
      effect: "deny",
    };`